.github/workflows/faucet.yml

```diff
@@ -0,0 +1,79 @@
+name: Faucet
+on:
+  repository_dispatch:
+    types: [sync-contracts]
+  push:
+    branches:
+      - main
```

Comment:
Remove this, this will cause this action to run on push, we only need to run this action in repo dispatch